test/functional/api/cas/cache.py

```diff
@@ -66,6 +65,20 @@ def get_status(self):
         status = self.get_statistics().config_stats.status.replace(' ', '_').lower()
         return CacheStatus[status]
 
+    @staticmethod
+    def get_metadata_max_size(cache_size, cache_line_size, number_of_cores=4096):
+        """Get memory size according to requirement """
+        constant = int(Size(100, Unit.MebiByte))
+
+        cache_line_count = int(cache_size.value) / int(cache_line_size.value)
+        per_cache_line_metadata = 68 + 2 * int(cache_line_size.value) / number_of_cores
```

Comment:
If `cache_size` and `cache_line_size` are Size's objects, then where do You check unit? These could be bytes or gibibytes. You should use the `get_value(Unit.MebiByte)` method rather than access their values directly (constant unit is MiB).